addons/crm/views/crm_lead_views.xml

```diff
@@ -576,47 +576,114 @@
             </field>
         </record>
 
+        <act_window name="Send email"
+                res_model="mail.compose.message"
+                binding_model="crm.lead"
+                binding_views="list"
+                view_mode="form"
+                target="new"
+                id="crm_lead_act_window_compose"
+                context="{
+                            'default_composition_mode': 'comment',
+                            'default_email_to':'{($object.email or \'\')|safe}',
+                            'default_use_template': True,
+                            'default_template_id': ref('crm.email_template_opportunity_mail'),
+                        }"
+        />
+    
         <!--
             crm.lead (as Opportunity) views
         -->
 
         <record id="crm_case_tree_view_oppor" model="ir.ui.view">
             <field name="name">crm.lead.tree.opportunity</field>
             <field name="model">crm.lead</field>
+            <field name="priority">20</field>
             <field name="arch" type="xml">
-                <tree string="Opportunities" decoration-muted="probability == 100" decoration-danger="activity_date_deadline and (activity_date_deadline &lt; current_date)">
+                <tree string="Opportunities" default_order="activity_date_deadline_my">
```

Comment:
Meh, really not sure the standard list view should have an order on activity_date_deadline_my as this is linked to an hack. I am not sure it will work, and I think we should limit the grouping on activity_date_deadline_my only to "my activities" menu item. To validate with LNA but strongly pushing to limit the use of activity_date_deadline_my to your specific action.